### PR TITLE
Test for 3.11 support

### PR DESCRIPTION
**Pull request by ahopkins**

### CODE REVIEW

**Comment by ahopkins** [comment]
I think there are some tweaks we need to make to the GH files. I honestly want to do bare minimum so we can rip them out next year.

File: .github/workflows/pr-linter.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         config:
-          - { python-version: 3.8, tox-env: lint}
+          - { python-version: 3.10, tox-env: lint}
```
Comment (ChihweiLHBird) [inline .github/workflows/pr-linter.yml:19]: Making `3.10` a `str` object, `"3.10"` may fix this PR.

Comment (ahopkins) [reply .github/workflows/pr-linter.yml:19]: Ahh, nice catch. Feel free to make the change and push it if you want.

Comment (ChihweiLHBird) [reply .github/workflows/pr-linter.yml]: Thank you, @ahopkins

**Comment by ChihweiLHBird** [comment]
> I think there are some tweaks we need to make to the GH files. I honestly want to do bare minimum so we can rip them out next year.

Why we will be able to remove them next year? Is there any successor of them?

**Comment by ahopkins** [comment]
Something much simpler like what sanic-ext uses. Plus I've been playing with getting rid of codecov and just using coverage.py and posting back the html.

**Pull request merged**